packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -48,6 +48,12 @@ export 'package:test_api/test_api.dart' hide
 /// Signature for callback to [testWidgets] and [benchmarkWidgets].
 typedef WidgetTesterCallback = Future<void> Function(WidgetTester widgetTester);
 
+/// The default refresh rate for continuous pumping
+const double kDefaultRefreshRate = 59.94;
+
+/// The default frame interval for continuous pumping
+const int kDefaultFrameIntervalInMicroseconds = 1E6 ~/ kDefaultRefreshRate;
```

Comment:
Why these values? 

These seem like they should probably be parameters passed to your new method, perhaps with defaults and a good explanation of what value you might want to set them to and why the default exists.

Reply:
In particular, I think these should not be publicly exposed, even if they're defined here.

Reply:
For why 59.94 Hz, [here](https://www.tvtechnology.com/news/whats-the-difference-between-5994fps-and-60fps#:~:text=In%20most%20cases%20%2260%22%20is,field%20rates%20such%20as%2060i.) and many other sources say it is what really 60Hz means in many devices, especially old ones, for historical reason of preventing resonance with city electricity. It is also what @dkwingsmt has been using in `pumpFrames`.

Reply:
I'm sharing these variables for referenced in unit test. It should be a better way. Should I just make it private and copy it to the unit test?